lib/workload/stateless/stacks/bssh-icav2-fastq-copy-manager/lambdas/query_bclconvert_outputs_handler/handler.py

```diff
@@ -0,0 +1,379 @@
+#     #     "manifest_b64gz": "H4sIADwI6GUC/+2cX2/bNhTFv0rR58kSL/W3b03argESpI3SdcAwELRE29pkSZPoLNmw7z7GrZlmy9Y0AWORuk85BgLk8nePDi8lOX8+rwp+AS98fw50MY9I7JVpBF6YpZHHy6LwUkoKAiItSgC/qteNxxteXw1i8IESQmL2MiBBBCwgCbCDtz8cn0Sv8h8TtiCEhxmjJFskoXdQ1IdtcyF6+ewiZMASLwHKw3AeeTAH4YVBmnjpHCIvg6LklJMoLDO/3chuI/3jdjn4H3nfVM1ymNXt8vmLZz/p0pMoi0S6AI9CSb2wSGKPUwi9IuZpVHKRJvNoWzrr+mrN+ysfAqD/X75P4yiLo/T6V8MAgoTPi3KzqcpPtTz/+btnFqE7ahYtYvtmbK/7vu3Rb98O7g0f5G+H7bqrhRQzeSmR3335nYmu7eXgvxLrTS0rRfCS5ZLLYVYMF3vEuKvLPpIvS95J0bMTIfuqQI4P5fh+w+tKXiHHR3I8akp1Tb9tu05NM+yw3TR4cT8YZs6vN5l8JYREhg9kuLjerVldDYjwsXvM4VVRC0zIR9I82zTbE8vlukZ+j5wdzytlSBwgH71hf76mTzdyNq8a5PgAjudtxz40vzbt7w074H3RlgJN+VCY/fbnbCUxIx91mtnmI27YD4P5afoe/GPeCEb8Y6ABocmNYHnIjoOAsDPC1I/ZdtKcLf/YI+T/Ktl26IEWLKefocO4oQdOQicI/enjBTBezEIHLVge2eF0cBI6IHSj0ONUC5YTK5yuSrYdeqYFy8EO6JmT0AGhP328AMaLIegfmlJI0a+rRpS3PrA8GHnM3Crd5QYANsBYA3YTZBizXvSb5h8fWZ5cNwHGehX8q3zX2wDYhqc4WFEtWB7bcZql1ns/u+39bOd9EtiRQZkjGZTc7kOy60NqRxsSN9qQRFqoS4BawV7V7CR1QOpPv98C7rdGrZ5GWiirh1YETBo5v7sC7q6jGDYBh829hRBgCJmlnmihqEd2RH9ifeaktzMn3WVOZsfBKnUkcu40P6D5jVDfTfZpqoWivr2dScfq+C9qdpI6IPX9Bj1g0D+F+WMtlPljOyIndpM6IHVDkfO1B+gjnivdeIB+jwYANsD4phsFtzbd3UeVQcSK45Wu3/YnKKEWij3Y8dwqdN/9gO7fm/sB3W905syIFiyHwIpJPyPWU4+0UNSpHdQjN6kDUt9DwgAmjFnqgRZqN83sSJjATeqA1I1Q3w2MQLRQCbN9Eh6O1etf1Gyt179222bEQePCbZt7NQCwAXvKHcDcMUsdtFDUIzvSHtykDkjdLHWqhaIe2+F16iZ1QOpmqd8IRT2xw+uhm9QBqZulHmmhqKd2eN36O5FppgXLSWrHW02Zm14H9LpZ6rEWinpmR8LEblIHpL6HXAfMdbNeT7RgOQ3sSJjETeqA1M1ST7VQ1IkdXk/dpA5I3Sz1TAtFHezwuvW7KQUtFHU7nmpQcNPrgF7fg9cBvW6WeqCFok7tSJjATeqA1M1SJ1oo6na8EUaJm9QBqZulTrVQ1O14R4BSN6kDUjdL/UYo6na8I0BDN6kDUjdLPdJCUbfjHQEauUkdkLpR6uGNUNTteFodhm5SB6Ru9jtiN4LlAHZ8M49aTx20UNSJHdTBTa8Den0PXgf0uqHd9GtfxxvxKOPG1/Hu0QDABhifJ+/4Tw+jn+Iz4iZ1QOpmqd8IloeWeD10kzogdaPU73i/evRet//96rupg6PU+00z+PNhWDHeDeANK6/r25Lt/vxRo4ak085/WVfLZi0aeSJkXxXD6UbO5lWzRw67ysyu+/W6q9R6ef1uxYeqWU5t+X3f9hNb82Hb96KQojyanNtPO3nt9ZO2FPXEln7AB6GWXk/wGr+UQh2Gy/OqFtNbes8LWbXNxBb+/vN6D66ux4iprPrdm+/7qpxYq4/WfDm163qCUfbm49uTCbp7F2QQ0GAqay54sRJ+KRZ8U8tixXs5zH4Z2jEs/FNpT9PyyQwpF9M7dp+JZTWo2WyCo9mRmsUvR7jmv/4GyMfqZFivAAA="  # pragma: allowlist secret
+#     # }
```

Comment:
Fair a bit of logic here and some from utils package. We should test this when we have more time a bit, down track. I can help with this. No action for now; early day.